great_expectations/datasource/fluent/pandas_dbfs_datasource.py

```diff
@@ -34,7 +34,8 @@ class PandasDBFSDatasource(PandasFilesystemDatasource):
     """Pandas based Datasource for DataBricks File System (DBFS) based data assets."""
 
     # instance attributes
-    type: Literal["pandas_dbfs"] = "pandas_dbfs"
+    # overridden from base `Literal['pandas_filesystem']`
+    type: Literal["pandas_dbfs"] = "pandas_dbfs"  # type: ignore[assignment] # base class has different type
```

Comment:
@Kilo59 It is interesting how previously, the "type: ignore" annotation was not needed...